Index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="Styles.css">
+    <title>Title</title>
+</head>
+<body>
+
+<header class="outer-container header-text">
+    <nav class="header-nav">
+        <ul>
+            <li><a href="/">Case studies</a></li>
+            <li><a href="/">Services</a></li>
+            <li><a href="/">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="inner-container header-text inner-container-header">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <button type="button">Let's work together</button>
+        </div>
+    <span class="image-header">
+    <img id="image header" src="assets/squiggle_line_stripe.svg" alt="squiggle line stripe"/>
```

Comment:
Je hebt dit icoontje nu als "echte" afbeelding ingevoegd, maar aangezien 'ie alleen van decoratieve waarde is en je hem absoluut moet positioneren, was een background-image ook een goede optie geweest!